site/app/templates/grading/electronic/NavigationBar.twig

```diff
@@ -7,6 +7,7 @@
                    data-href="{{ home_url }}"
                    id="main-page"
                 >
+                    <span hidden>Main Page</span>
```

Comment:
Is this for accessibility? I believe that the better option would be to use an [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute) attribute on the `<a>` tag, and then maybe also add `aria-hidden="true"` on the `<i>` tag.

```suggestion
                   id="main-page"
                   aria-label="Main Page"
                >
```

See https://fontawesome.com/v5.15/how-to-use/on-the-web/other-topics/accessibility.

@shailpatels can confirm on this.

Reply:
Good point. I had done that to make sure there weren't links with empty text, but I didn't realize aria-label would satisfy that condition.